packages/helpermodules/update_config.py

```diff
@@ -28,7 +28,7 @@
 
 
 class UpdateConfig:
-    DATASTORE_VERSION = 21
+    DATASTORE_VERSION = 22
```

Comment:
```suggestion
    DATASTORE_VERSION = 23
```
Es gab zwischenzeitlich schon einen PR, der den Datastore aktualisiert hat.